src/components/TypeLayout/TypeLayout.js

```diff
@@ -56,67 +47,76 @@ const TypeLayout = ({
   </StructuredListWrapper>
 );
 
-const TypeLayoutRow = ({ children, className, ...other }) => (
-  <StructuredListRow
-    className={classnames(`${namespace}__row`, className)}
+const TypeLayoutBody = ({ children, className, ...other }) => (
+  <StructuredListBody
+    className={classnames(`${namespace}__body`, className)}
     {...other}
   >
     {children}
-  </StructuredListRow>
+  </StructuredListBody>
 );
 
-const TypeLayoutBody = ({ children, className, ...other }) => (
-  <StructuredListBody
-    className={classnames(`${namespace}__body`, className)}
+const TypeLayoutCell = ({ children, className, ...other }) => (
+  <StructuredListCell
+    className={classnames(`${namespace}__cell`, className)}
     {...other}
   >
     {children}
-  </StructuredListBody>
+  </StructuredListCell>
 );
 
-const defaultProps = {
-  className: '',
-  children: null,
-};
+const TypeLayoutRow = ({ children, className, ...other }) => (
+  <StructuredListRow
+    className={classnames(`${namespace}__row`, className)}
+    {...other}
+  >
+    {children}
+  </StructuredListRow>
+);
 
 const propTypes = {
-  /** @type {string} Extra classes. */
-  className: string,
-
-  /** @type {ReactNode} Component children. */
+  /** Provide the contents of the node */
   children: node,
+
+  /** Provide an optional class to be applied to the containing node */
+  className: string,
 };
 
-TypeLayout.defaultProps = {
-  border: false,
-  bordered: null,
-  className: '',
+const defaultProps = {
   children: null,
-  size: 'md',
+  className: null,
 };
 
-const { children, className } = propTypes;
-
-const borderPropType = bool;
+const size = 'md';
```

Comment:
Hey I was wondering why this is abstracted. I'm seeing it only used in the propTypes and default propType below, right? To me this abstraction doesn't add anything, and just takes something away (i.e., immediately understanding what the default value is & what one of the `size` prop options are). Would you consider removing this & referencing `'md'` directly?

Reply:
Done!